templates/documentation/fire.html

```diff
@@ -11,6 +11,7 @@ <h2>Wildfire Data</h2>
     <li>Land cover types from 2015 were provided by the North American Land Change Monitoring System at a resolution of 30m.</li>
     <li>Historical fires from 1940&ndash;2021 were provided by the Alaska Interagency Coordination Center (AICC).</li>
     <li>Flammability projections are summarized across years 2000&ndash;2099 and were provided by ALFRESCO model outputs, simulated with the CRU TS 4.0 dataset (2000&ndash;2015) and NCAR-CCSM4 model outputs using the RCP 8.5 emissions scenario (2016&ndash;2099).</li>
+    <li>PM2.5 Concentration and AQI are model forecasts run by NASA once every 6 hours and available from the <a href="https://gmao.gsfc.nasa.gov/GMAO_products/NRT_products.php">GMAO GEOS NRT Data Products webpage</a>.</li>
```

Comment:
We're deriving the AQI, right?  So maybe this can instead say "Pm2.5 is from NASA, we derived AQI according to (algorithm)"?